.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "4"
```

Comment:
I guess we can do it later if/when loki will try to make it way into our 0.10 code. I think we can start using loki in simpler services first that use Node 4